### PR TITLE
Prepare crate for to crates.io publishing.

### PR DESCRIPTION
**Pull request by sandreim**
## Reason for This PR

Publish to crates.io.

## License Acceptance

By submitting this pull request, I confirm that my contribution is made under
the terms of the Apache 2.0 license.

## PR Checklist

`[Author TODO: Meet these criteria.]`
`[Reviewer TODO: Verify that these criteria are met. Request changes if not]`

- [x] All commits in this PR are signed (`git commit -s`).
- [x] The reason for this PR is clearly provided (issue no. or explanation).
- [x] The description of changes is clear and encompassing.
- [x] Any required documentation changes (code and docs) are included in this PR.
- [x] Any newly added `unsafe` code is properly documented.
- [x] Any user-facing changes are mentioned in `CHANGELOG.md`.

### CODE REVIEW

**Review by ioanachirca — APPROVED** [review]

File: Cargo.toml [hunk 1]
```diff
@@ -3,6 +3,10 @@ name = "versionize_derive"
 version = "0.1.1"
 license = "Apache-2.0"
 authors = ["Amazon Firecracker team <firecracker-devel@amazon.com>"]
+description = "Implements the Versionize derive proc macro."
+readme = "README.md"
+repository = "https://github.com/firecracker-microvm/versionize_derive"
+keywords = ["serialization", "version"]
```
Comment (ioanachirca) [inline Cargo.toml:10]: nit: maybe add `categories = ["encoding", "parsing"]` ?

Comment (sandreim) [reply Cargo.toml:10]: These are proper keywords. I think we need to dive a little deeper into this SEO thing, and optimize later.

**Review by acatangiu — APPROVED** [review]
nit: the two commits should've been squashed since the second is just reverting part of the first

**Comment by sandreim** [comment]
> nit: the two commits should've been squashed since the second is just reverting part of the first

Yeah, that makes sense.

**Pull request merged**